src/Config.php

```diff
@@ -356,6 +356,9 @@ protected function parseTaxonomy()
             if (!empty($taxonomy['options']) && is_array($taxonomy['options'])) {
                 $options = [];
                 foreach ($taxonomy['options'] as $optionKey => $optionValue) {
+                    if (is_numeric($optionKey)) {
+                        $optionKey = $optionValue;
+                    }
```

Comment:
Oh wait, that will need to go below the line, um, below this … as the value should be slugged.

Reply:
Tests say you were right :wink: